modules/tinymce/src/core/main/ts/fmt/ApplyFormat.ts

```diff
@@ -185,15 +207,15 @@ const applyFormat = function (ed: Editor, name: string, vars?: FormatVars, node?
           // Start wrapping
           if (!currentWrapElm) {
             // Wrap the node
-            currentWrapElm = dom.clone(wrapElm, false);
+            currentWrapElm = dom.clone(wrapElm, false) as Element;
```

Comment:
What type was this that it needs to be cast to an Element?

Reply:
`cloneNode` returns `Node` no matter what the original was. JS/TS has some fun quirks :roll_eyes: In saying that though, this is `DOMUtils.clone` so we could maybe update it to retain the type information.